src/Servers/Kestrel/test/InMemory.FunctionalTests/HttpsConnectionMiddlewareTests.cs

```diff
@@ -339,12 +339,14 @@ public async Task CertificatePassedToHttpContext(HttpProtocols httpProtocols)
             void ConfigureListenOptions(ListenOptions listenOptions)
             {
                 listenOptions.Protocols = httpProtocols;
-                listenOptions.UseHttps(new HttpsConnectionAdapterOptions
+                var options = new HttpsConnectionAdapterOptions
                 {
                     ServerCertificate = _x509Certificate2,
                     ClientCertificateMode = ClientCertificateMode.RequireCertificate,
-                    ClientCertificateValidation = (certificate, chain, sslPolicyErrors) => true
-                });
+                };
+
+                options.AllowAnyClientCertificate();
+                listenOptions.UseHttps(options);
```

Comment:
There's a lambda version of UseHttps that would be cleaner to use.

Reply:
Oh, that's much nicer